book/src/simple-payment-and-state-verification.md

```diff
@@ -0,0 +1,179 @@
+# Simple Payment and State Verification
+
+The transaction contains a signature. Validators store those signatures for a
+short period of time to ensure the transaction is not processed more than once.
+Validators provide a JSON RPC endpoint, which clients can use to query the
+cluster if a transaction has been recently processed. Validators also provide a
+PubSub notification, whereby a client registers to be notified when a given
+signature is observed by the validator. While these two mechanisms allow a
+client to verify a payment, they are not a proof and rely on a trusted
+validator.
+
+## Light Clients
+
+Light clients are cluster participants that are not running a fullnode.  The
+goal of this proposal is to provide a level of security that is greater than
+trusting a remote fullnode without requiring the light client to spend a lot of
+resources on verifying the ledger.
+
+The additional level of security for a light client depends on an initial
+canonical set of validators the light client considers to be the stakeholders of
+the cluster. As that set is changed, the client can update its internal set of
+known validators with [receipts](#receipts). This may become challenging with a
+large number of delegated stakes.
+
+Fullnodes themselves may want to use light client apis for performance reasons.
+For example, during the initial launch of a fullnode, the fullnode may use a
+cluster provided checkpoint of the state and verify it with a receipt.
+
+## Receipts
+
+The basic idea for receipts is to create a minimal hash chain from the
+transaction to a slice of PoH ledger that proves that the desired set of
+validators have voted on a block contain that transaction to the desired
+confirmation.
+
+Clients can examine this piece of the ledger and compute the finality using
+solana's fork selection rules.
+
+The receipts for both, state and payments, start with a Merkle path from the
+value into a bank-merkle that has been voted on and included in the ledger. A
```

Comment:
I would capitalize Bank-Merkle.
Generally, I think this would be easier to parse if capitalization of Merkle and merkle-types were consistent. Can you do a quick sweep and capitalize the others?